docs/Calendar/BookingsDiagnosticSummary.md

```diff
@@ -0,0 +1,50 @@
+| **-Identity**           | Booking MB SMTP Address (Only one per execution)                                                                                                                                                                                                                                                                                                                                  |
+| **-Staff**              | Verify Staff permissions for the Bookings mailbox.                                                                                                                                                                                                                                                                                                                                |
+| **-StaffMemberShipLog** | Get the Staff Membership Log for the Bookings mailbox.                                                                                                                                                                                                                                                                                                                            |
+| **-Graph**              | Use Graph API to get the Bookings mailbox, Staff, Services and Availiability.<br />Graph will allow the best compreehensive tests going through, as it will collect services data and staff, allowing to check more issues, such as permissions and more.<br />In the graph connection you will need the following scopes (Delegated):<br />User.Read.All<br />Bookings.Read.All |
+| **-MessageTrace**       | Get MessageTrace logs for the Bookings mailbox(Past 5 days).                                                                                                                                                                                                                                                                                                                      |
+| **-ExportToCSV**        | Export all data to CSV.                                                                                                                                                                                                                                                                                                                                                           |
+| **-ExportToExcel**      | Export the output to an Excel file with formatting.                                                                                                                                                                                                                                                                                                                               |
+
+---
+
+#### Syntax:
+
+Example to perform tests on a Bookings Mailbox:
```

Comment:
to perform ALL tests....

Reply:
Does this export by default to both CSV and Excel?

Reply:
yes, by default, all tests and exports are set to true.